raster/src/main/scala/geotrellis/raster/render/color/ColorClassifier.scala

```diff
@@ -429,9 +429,11 @@ object StrictColorClassifier {
     colorClassifier
   }
 
-  def fromQuantileBreaks(histogram: Histogram[Int], colors: Array[RGBA]) = {
-    val breaks = histogram.getQuantileBreaks(colors.length)
-    apply(breaks zip colors)
+  def fromQuantileBreaks[X <: AnyVal : ClassTag](histogram: Histogram[X], colors: Array[RGBA]) = {
```

Comment:
Not into this generic function that just checks two types.

I think if there were simply an overload that took `Array[Double], Array[RGBA]` (or traversables in general), and did the zip for you, that would be sufficient.